public/index.html

```diff
@@ -1,25 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8" />
-        <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta name="theme-color" content="#000000" />
-        <meta name="description" content="A list of browsers supported by TAO" />
-        <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-        <!--
-          Notice the use of %PUBLIC_URL% in the tags above.
-          It will be replaced with the URL of the `public` folder during the build.
-          Only files inside the `public` folder can be referenced from the HTML.
-
-          Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-          work correctly both with client-side routing and a non-root public URL.
-          Learn how to configure a non-root public URL by running `npm run build`.
-        -->
-        <title>Browsers supported by TAO</title>
-    </head>
-    <body>
-        <noscript>You need to enable JavaScript to run this app.</noscript>
-        <div id="root"></div>
-    </body>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="A list of browsers supported by TAO" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon-1-300x300.png" />
```

Comment:
Do we need a 300px by 300px favicon? 🤔 it is 16.7kb as compared to the 1.1kb.

Reply:
Replaced it by a 1.9-kb-still-looking-good version :-)